README.md

```diff
@@ -33,7 +34,16 @@ $ composer require pinkcrab/registerables
 
 ``` 
 
-## Example ##
+Once the module is included, we need to include the Registerable_Middleware Middleware. As this has its own dependencies, this will need to be added using construct_registration_middleware() from the App_Factory instance.
```

Comment:
Which module? 

Maybe go with a step-by-step  they need to follow, rather than "we need":

You need to include the module and the Registerable_Middleware. They come with their own dependencies which will need to be added using the construct_registration_middleware() from the App_Factory instance.